ons_alpha/bulletins/models.py

```diff
@@ -164,3 +175,16 @@ def previous_releases(self, request):
             context_overrides={"bulletins": previous},
             template="templates/pages/bulletins/previous_releases.html",
         )
+
+    @path("previous/v<int:version>/")
+    def previous_version(self, request, version):
+        try:
+            page_revision = Revision.objects.get(pk=version)
+        except Revision.DoesNotExist:
+            raise Http404 from None
+
+        return self.render(
+            request,
+            context_overrides={"page": page_revision.as_page_object()},
+            template="templates/pages/bulletins/bulletin_page.html",
```

Comment:
Nit: I don't think the template needs specifying here - it'll be the default for the page model.